configs/_base_/datasets/semi_partial_coco_detection.py

```diff
@@ -0,0 +1,146 @@
+# dataset settings
```

Comment:
what does the name partial stand for?

Reply:
Part of coco train2017 will be divided into labeled data, and the remaining data will be used as unlabeled data.

Reply:
than partial can be removed, semi_coco_detection is clear enough to indicate that the config is used for semi-supervised setting, essentially the whole dataset is used except that only part of labels is used.

Reply:
may indicate 1@10 in the file name, use semi-0.1_coco_detection.py